config/locales/defaults/en.yml

```diff
@@ -62,9 +63,9 @@ en:
       title: 'User satisfaction score'
       short_title: 'User satisfaction score'
       summary: 'Percentage of users who answered ''yes'' to the ''Is this page useful?'' survey'
-      context: 'Users who found the page useful, out of %{total_responses} responses'
+      context: 'Users who found the page useful, out of %{total_responses} %{response_suffix}'
       unit: ''
-      short_context: '%{total_responses} responses'
+      short_context: '%{total_responses} %{response_suffix}'
```

Comment:
I think I18n has inbuilt support for pluralisation like so:

```suggestion
      short_context: 
          one: '%{total_responses} response'
          other: '%{total_responses} responses'
```

Reply:
https://guides.rubyonrails.org/i18n.html#pluralization

Reply:
Oh nice, I'll sort it